UnityProject/Assets/Scripts/Messages/Client/RequestSyncMessage.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections;
+using UnityEngine.Networking;
+
+/// <summary>
+/// A message that is supposed to be sent from client when he's ready to accept sync data(transforms, etc.)
+/// Should probably be sent only once 
+/// </summary>
+public class RequestSyncMessage : ClientMessage<RequestSyncMessage>
+{
+	public static short MessageType = (short) MessageTypes.RequestSyncMessage;
+
+	public override IEnumerator Process()
+	{
+//		Debug.Log("Processed " + ToString());
+
+		yield return WaitFor(/*Subject, */SentBy);
+		//verify that the message isn't being abused here!
+//		Debug.Log("Requested sync");
+		CustomNetworkManager.Instance.SyncPlayerData(NetworkObject);
+	}
+
+//	public static RequestSyncMessage Send(/*GameObject subject*/)
+//	{
+//		var msg = new RequestSyncMessage{ /*Subject = subject.GetComponent<NetworkIdentity>().netId*/ };
+//		msg.Send();
+//		return msg;
+//	}
```

Comment:
unneeded(?), commented out code...

Reply:
Agree, I tend to.

Reply:
Well, without documentation it leads to a situation where no one wants to remove it... but no one knows why to keep it too. Not in the short term, but I like to think a little furter ahead...